src/main/java/uk/gov/hmcts/reform/pip/rules/rules/RulesService.java

```diff
@@ -33,4 +42,87 @@ public Publication getPublication(Integer publicationId) {
             throw new PublicationNotFoundException(String.format("Publication with ID %s not found", publicationId));
         }
     }
+
+    /**
+     * Method to get a court.
+     * @param courtId The ID of the publication to get.
+     * @return The court itself, if it passes the business rules and is found.
+     */
+    public CourtHearings getCourt(Integer courtId) {
+
+        Optional<CourtHearings> courtWithHearings = inMemoryRepository.getCourtHearings(courtId);
+        if (courtWithHearings.isPresent()) {
+            return courtWithHearings.get();
+        } else {
+            throw new CourtNotFoundException(String.format("Court with ID %s not found", courtId));
+        }
+    }
+
+    /**
+     * Method to get a list of courts.
+     * @param input The input search to get.
+     * @return The list of court itself, if it passes the business rules and is found.
+     */
+    public List<CourtHearings> getCourtList(String input) {
```

Comment:
none of these are rules and we shouldnt assume they are

Reply:
we've not been given any, as of today so im not sure what your 3 amigos brought out but if you could paste in what was said regarding the rules on this chat